pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -50,6 +50,13 @@ type GitHubRepositoryConfig struct {
 	// Whether we should show dashboard previews for pull requests.
 	// By default, this is false (i.e. we will not create previews).
 	GenerateDashboardPreviews bool `json:"generateDashboardPreviews,omitempty"`
+
+	// Path is the subdirectory for the Grafana data. If specified, Grafana will ignore anything that is outside this directory in the repository.
+	// This is usually something like `grafana/`. Trailing and leading slash are not required. They are always added when needed.
+	// The path is relative to the root of the repository, regardless of the leading slash.
+	//
+	// When specifying something like `grafana-`, we will not look for `grafana-*`; we will only look for files under the directory `/grafana-/`. That means `/grafana-example.json` would not be found.
+	Path string `json:"prefix,omitempty"`
```

Comment:
```suggestion
	Path string `json:"path,omitempty"`
```
this change will cascade though to the frontend client

Reply:
thanks!